public/index.php

```diff
@@ -6,13 +6,23 @@
 use Yiisoft\Yii\Web\ServerRequestFactory;
 
 require_once dirname(__DIR__) . '/vendor/autoload.php';
+require_once dirname(__DIR__) . '/src/globals.php';
```

Comment:
No, that is not desired since this way $container won't be available in `globals.php`.

Reply:
It's not used there.
Maybe `dd()` can be moved to `yiisoft/var-dumper` and initialized there?

Reply:
> It's not used there.

Part of the demo, one day, will become a template. It will be used in apps for certain.

> Maybe dd() can be moved to yiisoft/var-dumper and initialized there?

These shortcuts, while handy, aren't really a good practice. For now I'd avoid having these all over the place.

Reply:
> These shortcuts, while handy, aren't really a good practice. For now I'd avoid having these all over the place.

Then why `$container` should be available in `globals.php`?

Reply:
https://github.com/yiisoft/di/issues/99

Reply:
Looks like a part of `Yii2` design... 
And `Laravel`'s facades...

Reply:
Yes, kind of. That is hacky but handy.